pyBumpHunter/bumphunter_1dim.py

```diff
@@ -1429,19 +1903,44 @@ def print_bump_info(self):
         Information are printed to stdout.
         """
 
-        # Print stuff
+        # Start printing stuff
         print("BUMP WINDOW")
         print(f"   loc = {self.min_loc_ar[0]}")
         print(f"   width = {self.min_width_ar[0]}")
-        print(
-            f"   local p-value = {self.min_Pval_ar[0]:.5g}"
-        )
-        print(
-            f"   -ln(loc p-value) = {self.t_ar[0]:.5f}"
-        )
-        print(
-            f"   local significance = {norm.ppf(1 - self.min_Pval_ar[0]):.5f}"
-        )
+
+        # Check if there are multiple channels
+        if type(self.min_Pval_ar[0]) != np.ndarray:
```

Comment:
```suggestion
        if not isinstance(self.min_Pval_ar[0], np.ndarray):
```
But that seems still unstable to me, and again, why make single channel a special case?